electron/lib/remoteLog.js

```diff
@@ -0,0 +1,19 @@
+import Log from 'electron-log';
+import winston from 'winston';
+
+// Requiring `winston-papertrail` will expose
+// `winston.transports.Papertrail`
+require('winston-papertrail').Papertrail; // eslint-disable-line
+
+const winstonPapertrail = new winston.transports.Papertrail({
+  host: 'logs5.papertrailapp.com',
+  port: 43689
```

Comment:
we should also set here `program` to differentiate between cardano-node and daedalus output.